codec-stomp/src/test/java/io/netty/handler/codec/stomp/StompTestConstants.java

```diff
@@ -18,8 +18,8 @@
 public final class StompTestConstants {
     public static final String CONNECT_FRAME =
         "CONNECT\n" +
-            "host:stomp.github.org\n" +
             "accept-version:1.1,1.2\n" +
+            "host:stomp.github.org\n" +
```

Comment:
But we lose the ability to keep the insertion order, right? i.e. decoding an HTTP request and re-encoding it will result in different ordering.